yarn.lock

```diff
@@ -3801,7 +3801,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@gorhom/bottom-sheet@npm:4.4.7":
+"@gorhom/bottom-sheet@npm:^4":
   version: 4.4.7
```

Comment:
does this mean we're still using 4.4.7? maybe it was just an issue with not passing in snapPoints.value? 🤔